jms/1.7/modules/ROOT/pages/jms-connector-reference.adoc

```diff
@@ -770,10 +772,12 @@ Used to configure the RedeliveryPolicy#getMaximumRedeliveries()
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Context Factory a| Any |  |  |
-| Jndi Initial Factory a| String |  |  |
-| Jndi Provider Properties a| Object |  |  |
-| Jndi Provider Url a| String |  |  |
+| Context Factory a| Object | The fully qualified class name of the factory class that creates the initial context. |  |
```

Comment:
I wonder if we can just say: The fully qualified name of the factory class that creates the initial context. 

?